src/CNN/networks/layers/OwnDataset.py

```diff
@@ -25,7 +25,7 @@ def __getitem__(self, index):
             else:
                 data = file[str(data_idx) + '/data'][()][0]
             label = file[str(data_idx) + '/label'][()]
-            label = -np.log10(np.exp(-label))
```

Comment:
Same as above. Old code used log2 instead of log10 so I had to recalc this. So if one uses the new code for calculating the data, your change is correct. Thanks!